models/issue_comment.go

```diff
@@ -1148,6 +1148,66 @@ func fetchCodeCommentsByReview(e Engine, issue *Issue, currentUser *User, review
 	return pathToLineToComment, nil
 }
 
+// FetchCodeCommentsByLine fetches the code comments for a given treePath and line number
+func FetchCodeCommentsByLine(issue *Issue, currentUser *User, treePath string, line int64) ([]*Comment, error) {
+	opts := FindCommentsOptions{
+		Type:     CommentTypeCode,
+		IssueID:  issue.ID,
+		TreePath: treePath,
+		Line:     line,
+	}
+	var comments []*Comment
```

Comment:
make ?

Reply:
Thank you for the review. I will apply your suggestions. I don't understand what you mean by this comment though?

Reply:
Since we can't predict how many comments will be fetched from database, we can't give a meaningful capability on the make.

Reply:
I think he meant that comments variable needs to be initialized using make

Reply:
I think this is fine - AFAIU xorm will generate the comments array as appropriate in Find.